### PR TITLE
HTML-CSS-Popup task

### PR DESCRIPTION
**Pull request by kosh3n**
### HTML-CSS-Popup

[Demo](https://kosh3n.github.io/HTML-CSS-Popup/) |
[Code base](https://github.com/kosh3n/HTML-CSS-Popup)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

File: submissions/kosh3n/HTML-CSS-Popup/index.html [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+            <a href="#home"><img class="homo-img" src="https://img.icons8.com/parakeet/96/000000/experimental-home-parakeet.png" alt="home"></a>
+        </div>
+        <div class="welcome">
+            <p>Welcome to our site!</p>
+        </div>
+
+            <nav class="navigation">
+                <!-- icon">  -->
+                <label for="hamburger">
+                    <span tabindex="0"> <img src="dots-menu.png" alt="dots-menu.png"></span></label>
```
Comment (HannaSyn) [inline submissions/kosh3n/HTML-CSS-Popup/index.html:24]: For icons it's better set an empty alt attribute, read more [here](https://accessibility.psu.edu/images/imageshtml/#empty)

File: submissions/kosh3n/HTML-CSS-Popup/index.html [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+                <input type="checkbox" id="hamburger">
+                <ul class="nav-menu" tabindex="0">
+                    <!-- this section is shown -->
+                    <li><a href="#mail">
+                            <figure>
+                                <img src="https://img.icons8.com/stickers/100/000000/mail-contact.png" alt="mail">
+                                <figcaption>Mail</figcaption>
+                            </figure>
+                        </a></li>
+                    <li><a href="#">
```
Comment (HannaSyn) [inline submissions/kosh3n/HTML-CSS-Popup/index.html:34]:
```suggestion
                    <li>
                        <a href="#mail">
                            <figure>
                                <img src="https://img.icons8.com/stickers/100/000000/mail-contact.png" alt="mail">
                                <figcaption>Mail</figcaption>
                            </figure>
                        </a>
                    </li>
                    <li>
                        <a href="#">
```
Please, format your file to make it more readable

File: submissions/kosh3n/HTML-CSS-Popup/index.html [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+                                <figcaption>Apps</figcaption>
+                            </figure>
+                        </a></li>
+                    <li><a href="#">
+                            <figure>
+                                <img src="https://img.icons8.com/stickers/100/000000/speech-bubble.png" alt="Chat">
+                                <figcaption>Chat</figcaption>
+                            </figure>
+                        </a></li>
+                    <!-- and this section is hide by default -->
```
Comment (HannaSyn) [inline submissions/kosh3n/HTML-CSS-Popup/index.html:82]: Please, avoid redundant comments

File: submissions/kosh3n/HTML-CSS-Popup/index.html [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+        <div class="logo-conatiner">
+            <a href="#home"><img class="homo-img" src="https://img.icons8.com/parakeet/96/000000/experimental-home-parakeet.png" alt="home"></a>
+        </div>
+        <div class="welcome">
+            <p>Welcome to our site!</p>
+        </div>
+
+            <nav class="navigation">
+                <!-- icon">  -->
+                <label for="hamburger">
```
Comment (HannaSyn) [inline submissions/kosh3n/HTML-CSS-Popup/index.html:23]: Please, add possibility to open menu with keyboard

File: submissions/kosh3n/HTML-CSS-Popup/index.html [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+                            </figure>
+                        </a></li>
+                    <li><a href="#">
+                            <figure>
+                                <img src="https://img.icons8.com/stickers/100/000000/speech-bubble.png" alt="Chat">
+                                <figcaption>Chat</figcaption>
+                            </figure>
+                        </a></li>
+                    <!-- and this section is hide by default -->
+                    <li> <label for="show-more">
```
Comment (HannaSyn) [inline submissions/kosh3n/HTML-CSS-Popup/index.html:83]: Please, add possibility to view hidden icons with keyboard

File: submissions/kosh3n/HTML-CSS-Popup/style.css [hunk 6]
```diff
@@ -0,0 +1,202 @@
+body {
+    background-color: #dff6ff;
+    background-image: url("data:image/svg+xml,%3Csvg width='64' height='64' viewBox='0 0 64 64' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M8 16c4.418 0 8-3.582 8-8s-3.582-8-8-8-8 3.582-8 8 3.582 8 8 8zm0-2c3.314 0 6-2.686 6-6s-2.686-6-6-6-6 2.686-6 6 2.686 6 6 6zm33.414-6l5.95-5.95L45.95.636 40 6.586 34.05.636 32.636 2.05 38.586 8l-5.95 5.95 1.414 1.414L40 9.414l5.95 5.95 1.414-1.414L41.414 8zM40 48c4.418 0 8-3.582 8-8s-3.582-8-8-8-8 3.582-8 8 3.582 8 8 8zm0-2c3.314 0 6-2.686 6-6s-2.686-6-6-6-6 2.686-6 6 2.686 6 6 6zM9.414 40l5.95-5.95-1.414-1.414L8 38.586l-5.95-5.95L.636 34.05 6.586 40l-5.95 5.95 1.414 1.414L8 41.414l5.95 5.95 1.414-1.414L9.414 40z' fill='%230d97cd' fill-opacity='0.4' fill-rule='evenodd'/%3E%3C/svg%3E");
+    color: #06283D;
+}
+
+header {
```
Comment (HannaSyn) [inline submissions/kosh3n/HTML-CSS-Popup/style.css:7]: Please, avoid styling by tags [link](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)

File: submissions/kosh3n/HTML-CSS-Popup/style.css [hunk 7]
```diff
@@ -0,0 +1,202 @@
...
+}
+
+.welcome {
+    font-size: 2em;
+    padding-left: 300px;
+    padding-right: 100px;
+    text-align: center;
+}
+
+/*menu*/
```
Comment (HannaSyn) [inline submissions/kosh3n/HTML-CSS-Popup/style.css:28]: Get rid of unnecessary comments in code

File: submissions/kosh3n/HTML-CSS-Popup/style.css [hunk 8]
```diff
@@ -0,0 +1,202 @@
...
+    max-width: 220px;
+    margin: 1%;
+}
+
+img {
+    max-width: 220px;
+    max-height: 150px;
+}
+
+figcaption {
```
Comment (HannaSyn) [inline submissions/kosh3n/HTML-CSS-Popup/style.css:144]: Please, avoid styling by tags [link](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)

File: submissions/kosh3n/HTML-CSS-Popup/style.css [hunk 9]
```diff
@@ -0,0 +1,202 @@
...
+    cursor: pointer;
+    padding-top: 30px;
+    margin-left: 250px;
+}
+
+label:hover {
+    opacity: 0.5;
+}
+
+input {
```
Comment (HannaSyn) [inline submissions/kosh3n/HTML-CSS-Popup/style.css:109]: Please, avoid styling by tags [link](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)

**Review by HannaSyn — COMMENTED** [review]
Nice work! 
Please, see my comments below

**Comment by kosh3n** [comment]
I had rewrite code so now it should work properly now. Had deleted all comments and now the navigation from keybord is working. Please, check out my work again.

File: submissions/kosh3n/HTML-CSS-Popup/index.html [hunk 10]
```diff
@@ -0,0 +1,138 @@
...
+                <ul class="nav-menu">
+                    <li class="nav-item" tabindex="2"><a href="#mail" class="nav-link">
+                            <figure class="figure">
+                                <img class="menu-img" src="https://img.icons8.com/stickers/100/000000/mail-contact.png" alt="mail">
+                                <figcaption class="figcaption">Mail</figcaption>
+                            </figure>
+                        </a></li>
+                    <li class="nav-item " tabindex="2"><a href="#" class="nav-link">
+                            <figure class="figure">
+                                <img class="menu-img" src="https://img.icons8.com/stickers/100/000000/video.png" alt="video">
```
Comment (HannaSyn) [inline submissions/kosh3n/HTML-CSS-Popup/index.html:35]: For icons it's better set an empty alt attribute, read more [here](https://accessibility.psu.edu/images/imageshtml/#empty)
Please, check All icons in your project

**Review by HannaSyn — CHANGES_REQUESTED** [review]
Good, please, fix one comment below

**Comment by kosh3n** [comment]
Sorry, I was thinking that an empty alt attr should be only in first img with "logo", my mistake.  I fixed all alt in img, please, check out my work again.

**Review by artem-trubin — CHANGES_REQUESTED** [review]
Hi, there are some problems.

1. Focus on the popup button looks strange, see screenshot below. 
image
2. `More` button should hide after it is pressed. 
3. When the `More` menu is open it is possible to select the `More` twice while going through elements with the keyboard. But I guess this problem will disappear when you hide this button after resolving the previous problem.

**Pull request closed without merging**